Exiled.API/Features/Player.cs

```diff
@@ -838,7 +838,7 @@ public float Health
             get => healthStat.CurValue;
             set
             {
-                healthStat.CurValue = value;
+                ReferenceHub.playerStats.GetModule<HealthStat>().CurValue = value;
```

Comment:
This is actually wrong

Reply:
Whoops. I'll look over it.

Reply:
> This is actually wrong

I'm unable to look into this until Visual Studio stops being so dumb. But feel free to correct me.